mg400.repos

```diff
@@ -3,3 +3,7 @@ repositories:
     type: git
     url: git@github.com:HarvestX/PlayStation-JoyInterface-ROS2.git
     version: galactic
+  h6x_tf_manager:
+    type: git
+    url: git@github.com:HarvestX/h6x_tf_manager.git
```

Comment:
I want you to register https address
https://github.com/HarvestX/MG400_ROS2/pull/141